packages/block-library/src/video/style.scss

```diff
@@ -102,3 +87,18 @@
 .block-library-video-tracks-editor__add-tracks-container {
 	padding: $grid-unit-15;
 }
+
+.block-library-video-tracks-editor__single-track-editor .components-base-control {
```

Comment:
these look like editor specific styles so they should move to `editor.scss` right?

Reply:
Actually the whole changes on this file  should probably move there.

Reply:
I'm not sure how these changes will affect the whole of the editor. My intention was to specifically target the video tracks popover.